flutter-studio/src/io/flutter/android/AndroidModuleLibraryManager.java

```diff
@@ -204,7 +204,7 @@ private void doGradleSync(Project flutterProject, Function<Project, Void> callba
     if (dir == null) dir = flutterProject.getBaseDir().findChild(".android"); // For modules.
     assert (dir != null);
     EmbeddedAndroidProject androidProject = new EmbeddedAndroidProject(FileUtilRt.toSystemIndependentName(dir.getPath()), null);
-    androidProject.init();
+    androidProject.init(null);
```

Comment:
I assume this param is ok to be `null`?

Reply:
It is @Nullable.

Thanks for the quick review!